src/p5.dance.js

```diff
@@ -1116,6 +1120,29 @@ module.exports = class DanceParty {
     }, this);
   }
 
+  // Called when executing the AI block.
```

Comment:
Would it make sense to move this up where `setForegroundEffect`, `setBackgroundEffect` and `setBackground` are defined?

Reply:
It might do, though it also remains to be seen whether we use those functions or call an all new system.  I think we should revisit once we make a bit more progress.